GroupIS1024/Models/Team.cs

```diff
@@ -8,11 +8,12 @@ public class Team
 
         public int TeamId { get; set; }
 
+        [DisplayName("Team Name")]
         public string TeamName { get; set; }
 
-        public List<Coach> Coaches { get; set; }
+        public List<Coach>? Coaches { get; set; }
 
-        public List<Player> Players { get; set; }
+        public List<Player>? Players { get; set; }
```

Comment:
This same suggestion appeared in a previous code review.  When that occurs, first in wins.